drivers/exec/driver_test.go

```diff
@@ -870,10 +871,59 @@ func TestDriver_Config_validate(t *testing.T) {
 			}).validate())
 		}
 	})
+
+	t.Run("denied_host_ids", func(t *testing.T) {
+		invalidUidRange := "invalid denied_host_uids value"
+		invalidGidRange := "invalid denied_host_gids value"
+
+		for _, tc := range []struct {
+			uidRanges string
+			gidRanges string
+			errorStr  *string
+		}{
+			{uidRanges: "", gidRanges: "", errorStr: nil},
+			{uidRanges: "1-10", gidRanges: "1-10", errorStr: nil},
+			{uidRanges: "10-1", gidRanges: "", errorStr: &invalidUidRange},
+			{uidRanges: "", gidRanges: "10-1", errorStr: &invalidGidRange},
+		} {
+
+			validationErr := (&Config{
+				DefaultModePID: "private",
+				DefaultModeIPC: "private",
+				DeniedHostUids: tc.uidRanges,
+				DeniedHostGids: tc.gidRanges,
+			}).validate()
+
+			if tc.errorStr == nil {
+				require.Nil(t, validationErr)
+			} else {
+				require.Contains(t, validationErr.Error(), *tc.errorStr)
+			}
```

Comment:
switch to `shoenig/test` instead of `testify`

`must.NoError(t, err)`
`must.ErrorContains(t, err, "oops")`